tests/expand-manifest.jsonld

```diff
@@ -0,0 +1,326 @@
+      "input": "expand/st24-in.jsonld",
+      "expectErrorCode": "invalid annotation",
+      "requires": "JSON-LD*",
+      "option": {"specVersion": "json-ld*", "rdfstar": true}
+    }, {
+      "@id": "#tst25",
+      "@type": ["jld:NegativeEvaluationTest", "jld:ExpandTest"],
+      "name": "Invalid annotation node 5",
+      "purpose": "Node with @annotation on a list",
+      "input": "expand/st25-in.jsonld",
+      "expectErrorCode": "invalid annotation",
+      "requires": "JSON-LD*",
+      "option": {"specVersion": "json-ld*", "rdfstar": true}
+    }, {
+      "@id": "#tst26",
+      "@type": ["jld:NegativeEvaluationTest", "jld:ExpandTest"],
+      "name": "Invalid annotation node 6",
+      "purpose": "Node with @annotation on a list value",
+      "input": "expand/st26-in.jsonld",
+      "expectErrorCode": "invalid annotation",
+      "requires": "JSON-LD*",
+      "option": {"specVersion": "json-ld*", "rdfstar": true}
+    }, {
+      "@id": "#tst27",
+      "@type": ["jld:NegativeEvaluationTest", "jld:ExpandTest"],
+      "name": "Invalid annotation node 7",
+      "purpose": "Node with @annotation property on a top-level @included node is invalid",
+      "input": "expand/st27-in.jsonld",
+      "expectErrorCode": "invalid annotation",
+      "requires": "JSON-LD*",
+      "option": {"specVersion": "json-ld*", "rdfstar": true}
+    }, {
+      "@id": "#tst28",
+      "@type": ["jld:PositiveEvaluationTest", "jld:ExpandTest"],
+      "name": "Embedded annotation node 1",
+      "purpose": "Node with @annotation property on embedded subject",
+      "input": "expand/st28-in.jsonld",
+      "expect": "expand/st28-out.jsonld",
+      "requires": "JSON-LD*",
+      "option": {"specVersion": "json-ld*", "rdfstar": true}
+    }, {
+      "@id": "#tst29",
+      "@type": ["jld:PositiveEvaluationTest", "jld:ExpandTest"],
+      "name": "Embedded annotation node 1",
+      "purpose": "Node with @annotation property on embedded object",
+      "input": "expand/st29-in.jsonld",
+      "expect": "expand/st29-out.jsonld",
+      "requires": "JSON-LD*",
+      "option": {"specVersion": "json-ld*", "rdfstar": true}
+    }, {
+      "@id": "#tst30",
+      "@type": ["jld:NegativeEvaluationTest", "jld:ExpandTest"],
+      "name": "embedded node 11",
+      "purpose": "Embedded node with reverse relationship",
+      "input": "expand/st30-in.jsonld",
+      "expectErrorCode": "invalid embedded node",
+      "requires": "JSON-LD*",
+      "option": {"specVersion": "json-ld*", "rdfstar": true}
+    }, {
+      "@id": "#tst31",
+      "@type": ["jld:NegativeEvaluationTest", "jld:ExpandTest"],
+      "name": "embedded node 12",
+      "purpose": "Embedded node with expanded reverse relationship",
+      "input": "expand/st31-in.jsonld",
+      "expectErrorCode": "invalid embedded node",
+      "requires": "JSON-LD*",
+      "option": {"specVersion": "json-ld*", "rdfstar": true}
```

Comment:
This is till a single triple, so why forbid it?

Reply:
It is an issue in flattening/node map generation. As the values of `@id` don't go into a node map, the algorithm can't reach them to do the reversal. We could, of course, implement special logic to do that, but there's really no good use-case, IMHO.

Reply:
Fair enough :)